lib/download.ts

```diff
@@ -151,7 +144,7 @@ async function unzipVSCode(reporter: ProgressReporter, extractDir: string, extra
 			fs.mkdirSync(extractDir);
 		}
 
-		await spawnDecompressorChild('tar', ['-xzf', '-', '-C', extractDir], stream);
+		await spawnDecompressorChild('tar', ['-xzf', '-', '--strip-components=1', '-C', extractDir], stream);
```

Comment:
This just avoids needing to check and special case the nested directory (e.g. `VSCode-linux-arm64`, which is different for all platforms.)